textproc/riff/Portfile

```diff
@@ -24,15 +24,18 @@ checksums           ${distname}${extract.suffix} \
                     sha256  ba52c76c103f7e88301a61227b648d63115e52c7b14ff966073a8d0264f42bde \
                     size    495762
 
+patchfiles          patch-src-main-rs-moar-pager.diff
+
 destroot {
     xinstall -m 0755 \
         ${worksrcpath}/target/[cargo.rust_platform]/release/${name} \
         ${destroot}${prefix}/bin/
 }
 
-notes "
-    You can also install and use `moar` with `${name}`
-"
+notes {
+    You can also install and use `moar` with `${name}`. It will
+    also work with `moar +pager`.
```

Comment:
The change to this note isn't necessary if it will work with `moar` in all cases, `pager` variant or not.